client/src/constants.ts

```diff
@@ -108,8 +108,5 @@ export const TransactionSchema = yup.object().shape({
 });
 
 export const CreateBudgetSchema = yup.object().shape({
-  total: yup
-    .string()
-    .matches(/^\$(0|[1-9][0-9]{0,2})(,\d{3})*(\.\d{1,2})?$/)
-    .required("field is required"),
+  total: yup.string().required("field is required"),
```

Comment:
Do we need to change anything here? Or does the currency input handle all the validation for us?

Should, we add .number() back in and stuff?

Reply:
If we change to number, the formatting is no longer be valid. I think because we're putting in the $ as a prefix, it'll have to be read as a string. But the user still won't be able to input any letter characters because we're using the package for currency input.